packages/quantic/force-app/main/default/lwc/quanticNumberButton/quanticNumberButton.js

```diff
@@ -21,8 +21,8 @@ export default class QuanticNumberButton extends LightningElement {
    */
   @api selected;
 
-  get variant() {
-    return this.selected ? 'brand' : 'brand-outline';
+  get isPressed() {
```

Comment:
This property is not used.

Reply:
Used isPressed in the new implementation as conditional rendering for these btns is removed.